cachix/src/Cachix/Deploy/Activate.hs

```diff
@@ -8,11 +9,18 @@ import qualified Cachix.Client.NetRc as NetRc
 import qualified Cachix.Deploy.Websocket as CachixWebsocket
 import qualified Cachix.Types.BinaryCache as BinaryCache
 import Cachix.Types.Permission (Permission (..))
+import qualified Data.Aeson as Aeson
+#if MIN_VERSION_aeson(2,0,0)
```

Comment:
Do we have to support other versions of aeson?

Reply:
The backend is still running aeson-1

Reply:
Ah, I see! It consumes this as a library.